src/components/ui/header.tsx

```
@@ -2,36 +1,91 @@
+        <Link href="/" className="flex items-center space-x-3 rtl:space-x-reverse">
+          <Image
+            src="/img/merch-track-logo.png"
+            width={64}
+            height={64}
+            alt="MerchTrack Logo"
+          />
+          <span className="self-center text-2xl font-semibold whitespace-nowrap text-neutral-8 dark:text-neutral-1">MerchTrack</span>
+        </Link>
+        <div className="flex md:order-2 space-x-3">
+          <SignInButton mode='modal'>
+            <button className="text-primary bg-neutral-1  focus:ring-4 focus:outline-none focus:ring-accent-7 font-medium rounded-lg text-sm px-4 py-2 text-center hidden md:flex w-max outline outline-1 outline-primary hover:opacity-70">Sign In</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)